scripts/generate_json_docs.py

```diff
@@ -295,11 +296,14 @@ def build_type(type_id, title, contents):
 
 
 def clean_source_path(source):
```

Comment:
What is the point of this function? Please provide a docstring and then we can discuss the implementation. AFAICT the goal is to determine the true path to a module and there are better ways to do this that don't involve the `sys.path` guesswork.